python/nano/tutorial/notebook/training/tensorflow/run-nano-howto-guides-training-tensorflow-tests.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# export ANALYTICS_ZOO_ROOT=${ANALYTICS_ZOO_ROOT}
+# export NANO_HOME=${ANALYTICS_ZOO_ROOT}/python/nano/src
+export NANO_HOWTO_GUIDES_TEST_DIR=${ANALYTICS_ZOO_ROOT}/python/nano/tutorial/notebook/training/tensorflow
+
+set -e
+
+# the number of epoch to run is limited for testing purposes
+sed -i 's/epochs=10/epochs=1/' $NANO_HOWTO_GUIDES_TEST_DIR/*.ipynb
+
+# comment out the install commands
+sed -i 's/!pip install/#!pip install/' $NANO_HOWTO_GUIDES_TEST_DIR/*.ipynb
+
+# comment out the environment setting commands
+sed -i 's/!source bigdl-nano-init/#!source bigdl-nano-init/' $NANO_HOWTO_GUIDES_TEST_DIR/*.ipynb
+
+echo 'Start testing'
+start=$(date "+%s")
+
+# use nbconvert to test here; nbmake may cause some errors
+jupyter nbconvert --ExecutePreprocessor.timeout=600 --to notebook --execute ${NANO_HOWTO_GUIDES_TEST_DIR}/*.ipynb
```

Comment:
`jupyter nbconvert` is used here instead of `python -m pytest -s --nbmake --nbmake-timeout=600 --nbmake-kernel=python3 ${NANO_HOWTO_GUIDES_TEST_DIR}/*.ipynb` to avoid errors shown [here](https://github.com/intel-analytics/BigDL/actions/runs/3087422563/jobs/4992773410#step:6:1470):
![image](https://user-images.githubusercontent.com/54161268/191170020-d47ec7ad-4d05-4230-9a4c-f4dde02fc5f8.png)
![image](https://user-images.githubusercontent.com/54161268/191170088-0f7cd5bc-9ede-47f4-b826-851488619e2c.png)